CloudSQLReplicator/google/cloud/pontem/sql/replicator/util/storage.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2018 The Pontem Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Wrapper for Storage API client."""
+import logging
+
+import google.auth
+from google.cloud import storage
+
+from google.cloud.pontem.sql.replicator.util import gcp_api_util
+
+GCP_STORAGE_SCOPE = frozenset(
+    ['https://www.googleapis.com/auth/devstorage.read_write']
+)
+
+def build_storage_client(project=None, credentials=None):
+    """Builds Authorized Storage.Client with custom user agent.
+
+      Args:
+        project (str): Project ID.
+        credentials (google.auth.Credentials): credentials to authorize client.
+
+      Returns:
+        Storage.Client: returns authorized storage client with CloudSQL
+        Replicator user agent.
+    """
+
+    google.cloud._http.Connection.USER_AGENT = gcp_api_util._get_user_agent() # pylint: disable=protected-access
+    default_credentials, default_project = google.auth.default(
+        scopes=GCP_STORAGE_SCOPE
+    )
+    storage_client = (
+        storage.Client(project=project or default_project,
+                       credentials=credentials or default_credentials)
+    )
+
+    return storage_client
+
+
+def create_bucket(bucket_name,
+                  project=None,
+                  credentials=None):
+    """Creates a new bucket.
```

Comment:
seems that you have a duplicate comment.

You write: "Creates a new bucket." and then you write "Creates a new Cloud Storage Bucket."

Reply:
Removed duplicate comment.